### DISCUSSION TITLE
Provide complete samples, not code snippets?

### DISCUSSION THREAD

**Discussion by zhiming429438709** [fastapi/fastapi#7202] [Questions] [answered]
Provide complete samples, not code snippets

**Comment by Mause** [comment]
Please provide more details, this is not a question.

**Comment by ycd** [comment]
What do you mean by complete sample not snippet, what is the difference between them?

**Comment by Kludex** [comment, marked as answer]
If you're talking about the snippets in the documentation, you can find the full examples in the `docs_src` in this repository.

**Comment by tiangolo** [comment]
Thanks for the help here @Mause, @ycd, @Kludex!

@zhiming429438709 as the question is not complete or clear, I'm gonna close it. But if you still have questions, please create a new issue following the template.

> Sorry for the long delay! 🙈  I wanted to personally address each issue and they piled up through time, but now I'm checking each one in order.
